src/main/java/liuni/health/LiUniHealthCheck.java

```diff
@@ -0,0 +1,29 @@
+package liuni.health;
+
+import com.codahale.metrics.health.HealthCheck;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+
+public class LiUniHealthCheck extends HealthCheck {
+
+    public LiUniHealthCheck() { }
+
+    @Override
+    protected Result check() {
+        //If there needs to be more than verify credentials, expand this to run
+        //until either all results show healthy or return the first unhealthy result
+        return verifyCredentials();
+    }
+
+    private Result verifyCredentials() {
+        try {
+            Twitter twitter = TwitterFactory.getSingleton();
+            twitter.verifyCredentials();
```

Comment:
i do agree that verifyCredentials is good for a healthcheck.  In realworld scenario, twitter does have something called rate limiting on API calls.  Should the health still check the credentials?

Reply:
Ideally no, its better as a test. I just wasn't sure of a better health check for the moment.

Reply:
you're right that it isn't good. What would be a better health check then for the application then?